charts/opa-kube-mgmt/templates/deployment.yaml

```diff
@@ -181,7 +181,11 @@ spec:
             {{- end }}
             - --opa-url={{ .Values.useHttps | ternary "https" "http" }}://127.0.0.1:{{ .Values.port }}/v1
             - --opa-allow-insecure
+            {{- if has "*" .Values.mgmt.namespaces }}
```

Comment:
There is no need to introduce new if block, current code covers all cases already.